README.md

```diff
@@ -440,16 +440,18 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 4.4 Carefully choose your CI platform (Jenkins vs CircleCI vs Travis vs Rest of the world)
+## ![✔] 4.4 Avoid global test fixtures and seeds, add data per-test
 
-**TL;DR:** Your continuous integration platform (CICD) will host all the quality tools (e.g test, lint) so it should come with a vibrant ecosystem of plugins. [Jenkins](https://jenkins.io/) used to be the default for many projects as it has the biggest community along with a very powerful platform at the price of complex setup that demands a steep learning curve. Nowadays, it has become much easier to set up a CI solution using SaaS tools like [CircleCI](https://circleci.com) and others. These tools allow crafting a flexible CI pipeline without the burden of managing the whole infrastructure. Eventually, it's a trade-off between robustness and speed - choose your side carefully
+**TL;DR:** To prevent tests coupling and easily reason about the test flow, each test should add and act on its own set of DB rows. Whenever a test need to pull or assume the existence of some DB data - it must explicitly add that data and avoid mutating any other records
```

Comment:
Some small changes I'd suggest:
"Whenever a test need to pull or assume" -> "Whenever a test *needs* to pull or assume"

Reply:
> features like seeding tests before each run

What do you mean by that? @BrunoScheufler

Reply:
@BrunoScheufler Fixed and approved